packages/line/tests/Line.test.js

```diff
@@ -89,6 +90,44 @@ it('should create slice for each x value', () => {
     expect(slices.at(4).prop('slice').x).toBe(500)
 })
 
+it('should hide certain line charts by default given their ids', () => {
+    const data = [
+        {
+            id: 'A',
+            data: [
+                { x: 0, y: 3 },
+                { x: 1, y: 7 },
+                { x: 2, y: 11 },
+                { x: 3, y: 9 },
+                { x: 4, y: 8 },
+            ],
+        },
+        {
+            id: 'B',
+            data: [
+                { x: 0, y: 4 },
+                { x: 2, y: 8 },
+                { x: 3, y: 12 },
+                { x: 4, y: 10 },
+                { x: 5, y: 9 },
+            ],
+        },
+    ]
+    const wrapper = mount(
+        <Line 
+            width={500} 
+            height={300} 
+            data={data}  
+            enableSlices="x" 
+            animate={false} 
+            initialHiddenIds={['B']}
```

Comment:
I think we should also test that we can hide several series.

Reply:
unit test for hiding multiple series by default
<img width="800" alt="Screenshot 2024-07-11 at 9 13 04 PM" src="https://github.com/plouc/nivo/assets/10770556/70cd39dd-32b2-4924-a6ab-bc0a77e9244e">